jwst/tests_nightly/general/associations/test_sdp_pools.py

```diff
@@ -33,27 +39,28 @@ class AssociationBase(BaseJWSTTest):
 
     @property
     def pool_paths(self):
+        """Get the association pools"""
         if self._pool_paths is None:
             self._pool_paths = self.data_glob(self.test_dir, 'pools', glob='*.csv')
         return self._pool_paths
 
     @property
     def truth_paths(self):
+        """Get the truth associations"""
         if self._truth_paths is None:
             self._truth_paths = self.data_glob(*self.ref_loc, glob='*.json')
         return self._truth_paths
 
-asn_base = AssociationBase()
-try:
-    POOL_PATHS = asn_base.pool_paths
-except Exception:
-    POOL_PATHS = ['test will be skipped']
+ASN_BASE = AssociationBase()
+POOL_PATHS = ASN_BASE.pool_paths
```

Comment:
This line is causing the problem.  I think you still need the original `try/except` block around this.  Though this seems like a hacky way to do it.  Basically the way you have it set up bypasses the `@pytest.mark.bigdata` skipping mechanism on tests and test classes.  Though I'm not sure there's another way to do it.  Hmm.

Reply:
Oh, you are going to like this...